users/vieting/jobs/fairseq.py

```diff
@@ -191,4 +176,24 @@ def create_dict_ltr(self):
 Z 213
 """
         with open(self.out_dict_ltr_path.get(), 'w') as f:
-            f.write(dict_ltr_content)
\ No newline at end of file
+            f.write(dict_ltr_content)
+
+    
+    def get_common_dir(self):
+        """
+        Returns the common directory of all audios given in the corpora.
+        """
+        # TODO test this
```

Comment:
Is it tested now? Then please remove the comment.

Reply:
done!